docs/gateway-configuration/wifi-configuration-8021x.md

```diff
@@ -0,0 +1,47 @@
+# Wi-Fi 802.1x Configuration
+
+Enterprise Wi-Fi is currently only supported on gateways running our [generic profiles](/getting-started/install-kura/#installer-types). The following is a list of currently supported 802.1x configurations.
+
+ - TTLS-MSCHAPv2
+ - PEAP-MSCHAPv2
+ - TLS-EAP
```

Comment:
I was kinda expecting to find 3 section, one for each of the authentication methods. I have two instead, one of which combines TTLS and PEAP. It's fine but I would add a short sentence explaining that, since they are so similar, the documentation will explain how to use both of them at the same time.... or something along those lines.